### ISSUE TITLE
Default button Keros issue - Ensures button have discernible text

### ISSUE THREAD

**Issue by DhirajKhot** [OfficeDev/office-ui-fabric-react#11772]
### Environment Information

- **Package version(s)**: 6.211.1

### Describe the issue:
The split button has the Keros issue - 
button-name: Ensures buttons have discernible text

### Please provide a reproduction of the issue in a codepen:
Run Keros tool on below link - 
https://fabricweb.z5.web.core.windows.net/oufr/6.211.1/#/examples/button

![image](https://user-images.githubusercontent.com/20378231/72961957-ea382600-3dd8-11ea-9d08-8d7980673d37.png)

![image](https://user-images.githubusercontent.com/20378231/72961980-fcb25f80-3dd8-11ea-8a41-4df650b21b39.png)

Note - 
I could not repro this issue on latest v7. 
Can you please fix the issue on v6 as well? 
We are using v6 currently and have no immediate plans to upgrade to v7.

#### Actual behavior:
Keros is reporting errors on the split button component

#### Expected behavior:

No issues should be reported by Keros

**Comment by khmakoto** [comment]
Hi @DhirajKhot, thanks for filing this issue! I can reproduce it and I'll try to have a fix as soon as possible.
